numpy/linalg/lapack_lite/f2c_blas.c

```diff
@@ -380,7 +380,6 @@ static doublecomplex c_b1078 = {1.,0.};
     static logical nota, notb;
     static complex temp;
     static logical conja, conjb;
-    static integer ncola;
```

Comment:
This is generated code - it doesn't make any sense to:
* Run the static analyzer against it
* Change it based on the results of the analyzer